SampleApps/CarthageTest/CarthageTest/ViewController.swift

```diff
@@ -26,7 +26,7 @@ class ViewController: UIViewController {
         let payPalMessagingView = BTPayPalMessagingView(apiClient: apiClient)
         let threeDSecureClient = BTThreeDSecureClient(apiClient: apiClient)
         let venmoClient = BTVenmoClient(
-            apiClient: apiClient,
+            authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn",
```

Comment:
I think if we pull the latest from v7 we should have a constant for this here and in the SPMTest app